src/CalcViewModel/Common/LocalizationSettings.h

```diff
@@ -177,7 +177,7 @@ namespace CalculatorApp
 
             Platform::String^ GetEnglishValueFromLocalizedDigits(const std::wstring& localizedString) const
             {
-                if (m_resolvedName == L"en-US")
```

Comment:
Agree with @jlaanstra here.  Please update the type of `m_resolvedName` to be `wstring` and then the comparison will work here.